### ISSUE TITLE
Error on installing DWM due to X11

### ISSUE THREAD

**Issue by Sergiogd112** [ChrisTitusTech/linutil#225]
## Describe the bug
When installing DWM-Titus on POP OS it seems it does not install X11  libraries or it does not detect them on build. It also says that the distro is not supported when ubuntu and debian are. This is my first issue, so if there's anything I can improve or info that I should provide, feel free to tell me.
Here's my fastfetch output for context:
```
             /////////////                 ┌─────────── Hardware Information ───────────┐
         /////////////////////               󰌢   ROG Zephyrus G14 GA401IV_GA401IV (1.0)
      ///////*767////////////////               AMD Ryzen 7 4800HS (16) @ 2.90 GHz
    //////7676767676*//////////////             NVIDIA GeForce RTX 2060 Max-Q [Discrete]
   /////76767//7676767//////////////            AMD Renoir [Integrated]
  /////767676///*76767///////////////        󱄄   3840x2160 @ 60 Hz in 27″ [External] *
 ///////767676///76767.///7676*///////       󱄄   1920x1080 @ 120 Hz in 14″
/////////767676//76767///767676////////         11.98 GiB / 38.60 GiB (31%)
//////////76767676767////76767/////////    ├─────────── Software Information ───────────┤
///////////76767676//////7676//////////         Pop!_OS jammy 22.04 x86_64
////////////,7676,///////767///////////         Linux 6.9.3-76060903-generic
/////////////*7676///////76////////////         Mutter (X11)
///////////////7676////////////////////         bash 5.1.16
 ///////////////7676///767////////////     |──────────────Uptime / Age──────────────────|
  //////////////////////'////////////        OS Age    39 days
   //////.7676767676767676767,//////         Uptime    16 hours, 12 mins
    /////767676767676767676767/////        └────────────────────────────────────────────┘
      ///////////////////////////            ● ● ● ● ● ● ● ●
         /////////////////////
             /////////////

```
## To Reproduce
1.  Install DWM-Titus on Pop OS through linutil

## Expected behavior
It installs DWM-Titus
## Screenshots
![image](https://github.com/user-attachments/assets/569a71b9-0cd7-414a-aee7-e6afd2ab3ee9)

**Comment by fam007e** [comment]
I think you need to install header files... In your case, you need to do this: ```sudo apt-get -y install libx11-xcb1```. Also please raise issue in the `dwm-titus` repo as I believe thats where this came from.

**Comment by nnyyxxxx** [comment]
Will be resolved with the merge of #297
